index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  </head>
+
+  <body>
+
+    <header id="header">
+    <nav id="nav-bar">
+      <div class="logo"><img src="/images/logo.png" alt="" id="header-img"></div>
+      <div class="menu">
+        <ul>
+          <li><a class="nav-link" href="#services">Features</a></li>
+          <li><a class="nav-link" href="#video">How It Works</a></li>
+          <li><a class="nav-link" href="#price">Pricing</a></li>
+        </ul>
+      </div>
+    </nav>
+    </header>
+
+    <main>
+      <section id="top">
+        <h1>Handcrafted, home-made masterpieces</h1>
+       <form action="https://www.freecodecamp.com/email-submit" id="form">
+        <input type="email" id="email" name="email" placeholder="Enter Your Email" required>
+        <input type="submit" id="submit" value="Get Started">
+       </form>
+      </section>
+<!-- services Sections -->
+      <section id="services">
+        <div class="items">
+          <i class="fa-solid fa-fire"></i>
+          <div class="text">
+            <h3>Premium Materials</h3>
+          <p>Our trombones use the shiniest brass which is sourced locally. 
+            This will increase the longevity of your purchase.</p>
+          </div>
+        </div>
+    
+
+
+        <div class="items">
```

Comment:
I recommend that you remove extra spaces on your project. Removing it will makes your codes easier to read and makes it more professional.